spec/rubocop/cop/layout/trailing_empty_lines_spec.rb

```diff
@@ -30,67 +30,60 @@
     end
 
     it 'registers an offense for multiple trailing blank lines' do
-      offenses = inspect_source(<<~RUBY)
+      expect_offense(<<~RUBY)
         x = 0
 
+        ^{} 3 trailing blank lines detected.
 
 
       RUBY
-      expect(offenses.size).to eq(1)
-      expect(offenses.first.message).to eq('3 trailing blank lines detected.')
+
+      expect_correction("x = 0\n")
     end
 
     it 'registers an offense for multiple blank lines in an empty file' do
-      offenses = inspect_source(<<~RUBY)
+      expect_offense(<<~RUBY)
 
 
+        ^{} 3 trailing blank lines detected.
 
 
       RUBY
-      expect(offenses.size).to eq(1)
-      expect(offenses.first.message).to eq('3 trailing blank lines detected.')
+
+      expect_correction("\n")
     end
 
     it 'registers an offense for no final newline after assignment' do
       offenses = inspect_source('x = 0')
```

Comment:
There's another interesting place - the case doesn't work with `expect_offense`, because typing
```
expect_offense(<<~RUBY)
  x = 0
       ^ Final newline is missing
RUBY
```
Would actually have a newline after `x = 0` and will not cause an offense.  

I have some thoughts on how to work around it (possibly with another specific matcher), but I would like to leave it to a separate PR

Reply:
👍 😆

Reply:
That's perfect then.